lib/theme.json

```diff
@@ -11,6 +11,11 @@
 		},
 		"color": {
 			"background": true,
+			"gradient": true,
```

Comment:
Is this a new setting? It's not in the theme.json schema and we already have `customGradient` or `gradients`.